src/CommandLineParser.ts

```diff
@@ -232,13 +238,24 @@ function parseTSTLOptions(commandLine: ts.ParsedCommandLine, args: string[]): CL
     return { isValid: true, result: commandLine };
 }
 
-function getArgumentValue(argumentName: string, argument: string): ParseResult<string | boolean>
+function getArgumentValue(
+    argumentName: string,
+    argumentIndex: number,
+    args: string[]
+): ArgumentParseResult<string | boolean>
 {
+    const option = optionDeclarations[argumentName];
+    const argument = args[argumentIndex + 1];
+
+    if (option.type === "boolean" && (argument === undefined || argument.startsWith("-"))) {
+        // Set boolean arguments without supplied value to true
+        return { isValid: true, result: true, increment: 0 };
```

Comment:
Would it be better for `result` to be `option.default` here (in case a flag option is added that defaults to true if not specified)?

Reply:
I guess it should be `!option.default`? Otherwise setting the option without value would be the same as not setting it at all. I'm a little concerned that's just confusing. Would it ever make sense to have an option that is true by default and can be set to false by just adding its name to CLI? I'm not sure that is a valid use case.

Reply:
Yeah, actually you're probably right. If you set a flag up like that, it could just be inverted.